modules/nodes-sigstore-configure-image-policy.adoc

```diff
@@ -0,0 +1,203 @@
+<3> Defines a list of image scopes assigned to this policy.
+<4> Specifies the parameters that define how the images are verified.
+<5> Defines a root of trust for the policy.
+<6> Specifies the policy types that define the root of trust: here, a public key with Rekor verification.
+<7> For a public key policy, specifies a base64-encoded public key in the PEM format. The maximum length is 8192 characters.
+<8> Optional: Specifies a base64-encoded Rekor public key in the PEM format. The maximum length is 8192 characters.
+<9> Optional: Specifies one of the following processes to verify the identity in the signature and the actual image identity:
+* `matchRepoDigestOrExact` 
+* `matchRepository`.
+* `exactRepository`
+* `remapIdentity`
+<10> For `ExactRepository`, specify the repository with the image identity to be matched.
+--
++
+--
+.Example image policy object with a Fulcio certificate policy and the `matchRepository` match policy
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1alpha1
+kind: ImagePolicy <1>
+metadata:
+  name: p1
+  namespace: mynamespace <2>
+spec:
+  scopes: <3>
+    - example.com
+  policy: <4>
+    rootOfTrust: <5>
+      policyType: FulcioCAWithReko <6>
+      publicKey: <7>
+        fulcioCAData: a2V5RGF0YQ==
+        fulcioSubject:
+          oidcIssuer: "\https://expected.OIDC.issuer/"
+          signedEmail: "\expected-signing-user\@example.com"
+        rekorKeyData: cmVrb3JLZXlEYXRh <8>
+    signedIdentity:
+      matchPolicy: exactRepository <9>
+      exactRepository:
+        repository: example.com/repository/image <10>  
+----
+<1> Creates an `ImagePolicy` object.
+<2> Specifies the namespace where the image policy is applied. 
+<3> Defines a list of image scopes assigned to this policy.
+<4> Specifies the parameters that define how the images are verified.
+<5> Defines a root of trust for the policy.
+<6> Specifies the policy types that define the root of trust. Here, a Fulcio certification with required Rekor verification.
+<7> For a Fulcio certificate policy, the following parameters are required:
+* `fulcioCAData`: Specifies a base64-encoded Fulcio certificate in the PEM format. The maximum length is 8192 characters.
+* `fulcioSubject`: Specifies the OIDC issuer and the email of the Fulcio authentication configuration.
+<8> Specifies a base64-encoded Rekor public key in the PEM format. This parameter is required when when the `policyType` is `FulcioCAWithReko`. The maximum length is 8192 characters.
+<9> Optional: Specifies one of the following processes to verify the identity in the signature and the actual image identity:
+* `matchRepoDigestOrExact` 
+* `matchRepository`.
+* `exactRepository`
+* `remapIdentity`
+<10> For the `exactRepository` match policy, specifies the repository that contains the image identity and signature.
+--
+
+. Create the cluster image policy object:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
```

Comment:
I am not sure if this refers to the PublicKey or the FulcioCAWithRekor example above. I reviewed the example output below, assuming it refers to the PublicKey ImagePolicy.